projectDocs/dev/developerGuide/developerGuide.t2t

```diff
@@ -781,20 +781,31 @@ This manifest file contains key = value pairs declaring info such as the add-on'
 Although it is highly suggested that manifests contain all fields, the fields marked as mandatory must be included.
 Otherwise, the add-on will not install.
 
-- name (string): A short unique name for the add-on. This is used to differentiate add-ons internally and is not shown to the user. (Mandatory)
-- summary (string): The name of the add-on as shown to the user. (Mandatory)
-- version (string): The version of this add-on; e.g. 2.0. (Mandatory)
-- author (string): The author of this add-on, preferably in the form Full Name <email address>; e.g. Michael Curran <mick@kulgan.net>. (Mandatory)
+- name (string, required): A short unique name for the add-on.
+This is used to differentiate add-ons internally and is not shown to the user.
+The recommended convention is lowerCamelCase.
+Special characters should be avoided.
```

Comment:
Please consider noting here whether spaces are permitted. In fact, it may be best to define which "special characters" can/can't be used:
```
-
_
SPACE
period
```
being some of the more worth-while commenting upon.